ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -421,8 +421,8 @@ - (NSDictionary *)powerState
 {
 #if RCT_DEV && !TARGET_OS_TV && !TARGET_IPHONE_SIMULATOR
     if ([UIDevice currentDevice].batteryState == UIDeviceBatteryStateUnknown) {
-        RCTLogWarn(@"Battery state is `unknown` which could mean battery monitoring is disabled." +
-                    "You may need to enable monitoring with `[UIDevice currentDevice].batteryMonitoringEnabled = TRUE`");
+        RCTLogWarn(@"Battery state is `unknown` which could mean battery monitoring is disabled. "
```

Comment:
isn't the `+` needed for concatenation?

Reply:
That's what I thought too...

<img width="872" alt="Screen Shot 2019-03-26 at 11 56 59 AM" src="https://user-images.githubusercontent.com/4356436/55017234-4d410c80-4fbe-11e9-870b-be611d158953.png">

Reply:
Also see: https://github.com/facebook/react-native/blob/master/React/Base/RCTModuleData.mm#L65